src/test/java/io/reactivex/flowable/FlowableNullTests.java

```diff
@@ -1341,6 +1341,7 @@ public void flatMapNotificationOnErrorNull() {
     }
 
     @Test(expected = NullPointerException.class)
+    @Ignore("No longer crashes with NPE but signals it; tested elsewhere.")
```

Comment:
I had two options: I remove it and I get a question on why or ignore it with comments and get a question why...